### ISSUE TITLE
pyosmium-get-changes seems to stuck from time to time

### ISSUE THREAD

**Issue by stevo01** [osmcode/pyosmium#143]
Hi all,
I am using pyosmium-get-changes for update of osm database and call it inside a [replication script](pyosmium-get-changes).
The replication script is called every five minutes from cron.
The pyosmium-get-changes script hangs after several days or sometimes weeks. 
used version: python3-pyosmium/now 2.15.1-1 amd64 [installed,local]

The logfile looks like:
```
[INFO ] 2020-10-11.00:00:01 start update: osmosis replication is 5 minutes behind upstream
[INFO ] 2020-10-11.00:00:01 make backup copy of sequence_file
[INFO ] 2020-10-11.00:00:01 bgn: pyosmium-get-changes sequence=4233267
[INFO ] 2020-10-11.00:00:02 end: pyosmium-get-changes sequence=4233272
[INFO ] 2020-10-11.00:00:02 pyosmium-get-changes passed /replication/work/data/replicate-1602374401.osc
[INFO ] 2020-10-11.00:00:02 skip merging with existing diff
[INFO ] 2020-10-11.00:00:02 start osm2pgsql
[INFO ] 2020-10-11.00:03:12 osm2pgsql passed
[INFO ] 2020-10-11.00:03:12 end update: osmosis replication is 0 minutes behind upstream
[INFO ] 2020-10-11.00:03:12 end replication Sun Oct 11 00:03:12 UTC 2020
[INFO ] 2020-10-11.00:05:02 bgn replication Sun Oct 11 00:05:01 UTC 2020
[INFO ] 2020-10-11.00:05:02 start update: osmosis replication is 5 minutes behind upstream
[INFO ] 2020-10-11.00:05:02 make backup copy of sequence_file
[INFO ] 2020-10-11.00:05:02 bgn: pyosmium-get-changes sequence=4233272
```

I start the script with option -v
```
pyosmium-get-changes -s 500 -f $SEQFILE -o $THISFILE -v 2>&1 | tee -a $GETCHANGES_LOGFILE
```
The logfile content:
```
INFO: Using replication server at https://planet.osm.org/replication/minute/
```

Any idea whats the reason and/or how to get more debug infos from the script?

**Comment by lonvia** [comment]
What version of pyosmium do you use?

**Comment by stevo01** [comment]
root@osmdbutils:/replication/work# apt list --installed | grep "pyosm"
python3-pyosmium/now **2.15.1**-1 amd64 [installed,local]

**Comment by lonvia** [comment]
This issue has most likely been solved by df0929e66f350c2b5862a6ab79ca32340f3129cf which is included in the latest release. For Debian you can easily install the latest version via [Debian Backports](https://backports.debian.org/Instructions/).

**Comment by stevo01** [comment]
Thanks for fast feedback. I made update to 3.0.1 and restart my update procedure.
